externs/shaka/abr_manager.js

```diff
@@ -96,6 +96,15 @@ shaka.extern.AbrManager = class {
    */
   segmentDownloaded(deltaTimeMs, numBytes, allowSwitch) {}
 
+  /**
+   * Notifies the ABR that it is a time to suggest new streams. This is used by
+   * the Player when it finishes adding the last partial segment of a fast
+   * switching stream.
+   *
+   * @exportDoc
+   */
+  allowSuggestStreams() {}
```

Comment:
Overall, I think the design of this PR looks fine, but I don't like the name of this method.

`allowSuggestStreams` sounds to me like it's setting a configuration value, like it will make the abr manager start suggesting streams from now on.

To better convey that this will trigger at most one stream suggestion, I'd go for a name more like `trySuggestStreams`.

Reply:
Changed!